keps/sig-cloud-provider/2436-controller-manager-leader-migration/README.md

```diff
@@ -323,10 +327,47 @@ does not change incompatibly across those versions.
 
 Version skew is handled as long as the leader name is consistent across all control plane nodes during upgrade.
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [X] Other
+  - Describe the mechanism: this feature must be explicitly enabled by `--enable-leader-migration` flag
```

Comment:
I'd like to have a featuregate in addition to this flag so that callers in 1.21 have a clear understanding of the stability level feeding this flag.  I see this as comparable to kube-apiserver requiring a featuregate and a flag to enable a new alpha api.

Reply:
Sounds reasonable. Thank you.